src/EnergyPlus/IOFiles.cc

```diff
@@ -140,6 +140,7 @@ std::ostream::pos_type InputFile::position() const noexcept
 void InputFile::open(bool, bool)
 {
     file_size = fs::file_size(filePath);
+    // basic_fstream is a template, it has no problem with wchar_t (which filePath.c_str() returns on Windows)
```

Comment:
I added one comment here since that was one use of c_str() that wasn't obviously ok.